### PR TITLE
feat: vercast 1.0.0

### PR DESCRIPTION
**Pull request by gleich**
Signed-off-by: Matt Gleich <git@mattglei.ch>

## Description

View your vercel deployments right from raycast. It even refreshes every second to show the latest changes!

## Type of change

New extension

## Screenshot

example

## Checklist

- [x] I read the [contribution guidelines](contributing.md)

### CODE REVIEW

**Comment by thomaspaulmann** [comment]
> Is the failing CI because I'm using yarn?

Might be the case. We rely on `npm` for now. Could you switch? Same for your other extension #21

File: extensions/vercast/package.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
+  "name": "vercast",
+  "title": "vercast",
+  "description": "View your vercel deployments",
+  "icon": "vercel-icon.png",
+  "author": "matt",
```
Comment (fe9lix) [inline extensions/vercast/package.json:6]: Please also add this field to the manifest:
`"license": "MIT"`

**Review by fe9lix — CHANGES_REQUESTED** [review]
Cool! Last change request and I think then we're ready to merge:
I currently only see "Vercast" in root search, so when I search for "vercel" I won't find your command. I would suggest to put "Search Vercel Deployments" as command title instead of description.

**Comment by fe9lix** [comment]
@gleich It's merged now and available in the store. Something for your next update maybe:

Extension name: "Vercel Deployments"
Command name: "Search Deployments", subtitle: "Vercel"
Description: "Quickly open and check status of your Vercel deployments."

I know that those properties can be a bit confusing...

**Pull request merged**